roles/custom/matrix-client-element/defaults/main.yml

```diff
@@ -135,3 +135,21 @@ matrix_client_element_configuration_extension: "{{ matrix_client_element_configu
 # Holds the final Element configuration (a combination of the default and its extension).
 # You most likely don't need to touch this variable. Instead, see `matrix_client_element_configuration_default`.
 matrix_client_element_configuration: "{{ matrix_client_element_configuration_default | combine(matrix_client_element_configuration_extension, recursive=True) }}"
+
+# Enable Element Location sharing functionality
+# More info: https://element.io/blog/element-launches-e2ee-location-sharing/
+# How to host your own map tile server: https://matrix.org/docs/guides/map-tile-server
+matrix_client_element_location_sharing: false
+matrix_client_element_location_map_style: "{{ lookup('template', 'templates/style.json.j2') }}"
```

Comment:
In the future we can consider adding a `matrix_client_element_location_map_style_extension`, which gets merged with the base template and provides a way for people to override some parts of the configuration easily. We use this approach in various roles.

See:

- `matrix_client_element_configuration_default`
- `matrix_client_element_configuration_extension_json`
- `matrix_client_element_configuration_extension`
- `matrix_client_element_configuration`

I wonder if we should rename `matrix_client_element_location_map_style` to `matrix_client_element_location_map_style_default` in preparation for this.

.. but.. if you'd like, you can keep things as they are. We probably shouldn't be overcomplicating things.

Reply:
Done. I have implemented this functionality for location sharing map style.